common/src/main/java/io/github/gaming32/bingo/BingoCommand.java

```diff
@@ -517,6 +543,39 @@ private static int resetGame(CommandContext<CommandSourceStack> context) {
         return Command.SINGLE_SUCCESS;
     }
 
+    private static int resign(CommandSourceStack source) throws CommandSyntaxException {
+        if (Bingo.activeGame == null) {
+            throw NO_GAME_RUNNING.create();
+        }
+        ServerPlayer player = source.getPlayerOrException();
+        BingoBoard.Teams team = Bingo.activeGame.getTeam(player);
+        if (!team.any()) {
```

Comment:
This allows you to resign your entire team, even if you're not the only member. Is that desired?

Reply:
Yes, it's pretty hard to accidentally resign. I'd expect teams to coordinate if they want to resign. It's already possible to sabotage your own team by completing never goals